driftbench/data_generation/sample.py

```diff
@@ -36,14 +51,26 @@ def _generate_latent_information(dataset_specification, rng, x_scale, y_scale):
     latent_information = []
     for i in range(N):
         # Apply some random noise on the base values
-        x0 = base_latent_information.x0 + rng.normal(size=len(base_latent_information.x0), scale=x_scale)
-        y0 = base_latent_information.y0 + rng.normal(size=len(base_latent_information.y0), scale=y_scale)
+        x0 = base_latent_information.x0 + rng.normal(
```

Comment:
Any idea to remove the code duplicates? We already had them, but now it gets more nasty...

One idea may be to have  a `latent` dict which keys `y0`, `x1`, like:


```python

latent["x0"] =  getattr(base_latent_information, 'x0') + rng.normal(
             size=len(getattr(base_latent_information, 'x0')), scale=scale["x"]
         )
```

What do you think?